docs/pages/machine-id/reference/gitlab.mdx

```diff
@@ -62,10 +58,18 @@ spec:
     allow:
         # project_path restricts joins to jobs that originate within the
         # specified project.
+        #
+        # This field supports simple "glob-style" matching:
```

Comment:
```suggestion
        # This field supports glob-style matching:
```
Since we're already saying "glob-style", I don't think we need to qualify it further (so we can remove "simple"), and we don't need to quote "glob-style". This also applies to the other instances of this comment.

Reply:
613a1c2c7b